app/scripts/presenters/visit_presenter.js

```diff
@@ -3,12 +3,20 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 define([
-  'underscore'
-], function (_) {
+  'underscore',
+  'lib/image_proxy'
+], function (_, imageProxy) {
   'use strict';
 
   var MINIMUM_IMAGE_ENTROPY = 1.75;
-  var MINIMUM_IMAGE_WIDTH = 270;
+  var IMAGE_DISPLAY_WIDTH = 270;
+  var IMAGE_DISPLAY_HEIGHT = 180;
```

Comment:
Not sure if we use these constants elsewhere, or if they're candidates for moving into the Convict config file. I see that we're also hardcoding a "270px" in the Sass files below, so convict wouldn't help with that.